airbyte-integrations/connectors/source-woocommerce/unit_tests/integration/utils.py

```diff
@@ -16,12 +16,13 @@ def catalog(stream_name: str, sync_mode: SyncMode) -> ConfiguredAirbyteCatalog:
     return CatalogBuilder().with_stream(stream_name, sync_mode).build()
 
 
-def config() -> ConfigBuilder:
-    return ConfigBuilder()
-
-
-def source() -> SourceWoocommerce:
-    return SourceWoocommerce()
+def config() -> MutableMapping[str, Any]:
+    return {
+        "api_key": "test_api_key",
+        "api_secret": "test_api_secret",
+        "shop": "airbyte.store",
+        "start_date": "2017-01-01",
+    }
 
 
 def read_output(
```

Comment:
You can scrap this and catalog methods with the new reading utils, correct?

Reply:
yes, I'll cleanup unused things ...